docs/guides/using-requirements-file.rst

```diff
@@ -0,0 +1,68 @@
+Using a Requirements File
+=========================
+
+The requirements file option is useful for specifying dependencies required for building the documentation. Additional uses specific to Read the Docs are mentioned at the end of this guide.
+
+For details about the purpose of pip requirements file and how to create one, check out `pip user guide`_  
+
+.. note:: For the purpose of building your documentation with RTD, *project* is the documentation project, and *project root* is the directory where all the documentation is stored, often named ``docs``. 
+
+To use the requirements file, create and place the requirements file in the root directory of your documentation directory. For example::
+
+    docs/requirements.txt
+
+Specifying a requirements file
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Using the YAML configuration file
+---------------------------------
+The recommended approach for specifying a pip requirements file is to use a YAML (.yml) file. 
```

Comment:
Instead of "a YAML file", I'd suggest "a readthedocs.yml file", perhaps with a ref link to our YAML config page, at http://docs.readthedocs.io/en/latest/yaml-config.html